packages/notifier/src/publish-kit.js

```diff
@@ -216,75 +216,90 @@ const initDurablePublishKitState = (options = {}) => {
  */
 const getEphemeralKey = facets => facets.publisher;
 
-/** @type {WeakMap<DurablePublishKitEphemeralKey, {currentP, tailP, tailR}>} */
+/**
+ * @typedef DurablePublishKitEphemeralData
+ * @property {Promise<*> | undefined} currentP The current-result promise
+ *   (undefined unless resolved with unrecoverable ephemeral data)
+ * @property {Promise<*>} tailP The next-result promise
+ * @property {import('@endo/promise-kit').PromiseKit<*>['resolve'] | undefined} tailR The next-result resolver
```

Comment:
Why import the resolve definition? It's pretty standard:

```suggestion
 * @property {((value: *) => void) | undefined} tailR The next-result resolver
```

Reply:
Done.